doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -536,7 +536,7 @@ nRF RPC libraries
 Other libraries
 ---------------
 
-|no_changes_yet_note|
+* Removed the following unused SDFW services: ``echo_service``, ``reset_evt_service``, and ``sdfw_update_service``.
```

Comment:
Shouldn't this entry be moved in this page? https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/abi_compatibility.html

Reply:
It will appear there as well when the SDFW version that removes them is released. The comment in this changelog is referring to the removal of the client side implementation of the services.